### ISSUE TITLE
collapseWhitespace option adds unwanted spaces to javascript strings

### ISSUE THREAD

**Issue by antelle** [kangax/html-minifier#513]
``` bash
$ npm install html-minifier@1.2.0
$ ./node_modules/.bin/html-minifier --remove-comments --collapse-whitespace app.html -o min.html && cat min.html | grep -oh '...<?xml'
 ' <?xml
```

Original file and result attached.
Contents before:

``` html
<!DOCTYPE html>
<html>
<head lang="en">
    <script   >
var xml = '<?xml version="1.0" encoding="utf-8"?>'
</script>
</head>
</html>
```

Contents after:

``` html
<!DOCTYPE html><html><head lang="en"><script>var xml = ' <?xml version="1.0" encoding="utf-8"?> '</script></head></html>
```

v1.1.1 was ok

[Archive.zip](https://github.com/kangax/html-minifier/files/149931/Archive.zip)

**Comment by alexlamsl** [comment]
Fixed by #509

```
<!DOCTYPE html><head lang=en><script>var xml="<?xml version="1.0" encoding="utf-8"?>"</script>
```

**Comment by antelle** [comment]
Thank you! Please push a new release to npm
